wire/modules/Inputfield/InputfieldPageAutocomplete/InputfieldPageAutocomplete.module

```diff
@@ -360,7 +360,7 @@ _OUT;
 			if(!$f->type instanceof FieldtypeText) continue; 
 			$notes .= ' ' . $f->name . ','; 
 		}
-		$field->notes = rtrim($notes, ','); 
+		$field->notes = rtrim($notes, ',') . '; Example: parent.title title'; 
```

Comment:
I would change that to 

```
$field->notes = rtrim($notes, ',') . '; ';
$field->notes .= __('Example');
$field->notes .= ': parent.title title';
```
to make it translatable. I don’t know if that makes sense.